castor-service/src/main/java/io/carbynestack/castor/service/download/DefaultTuplesDownloadService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 - for information on the respective copyright owner
+ * Copyright (c) 2024 - for information on the respective copyright owner
```

Comment:
Copyright must be in format 
> ...
> Copyright (c) 2021 - 2024 - for information on the respective copyright owner
> ...

-> Copyright (c) [Year of file added] - [Year last modified] ...
if [Year of file added] is the same as [Year last modified] it is simply
-> Copyright (c) [Year of file added] ...